winml/lib/Api.Image/TensorToVideoFrameConverter.cpp

```diff
@@ -483,7 +483,8 @@ void TensorToVideoFrameConverter::ConvertGPUTensorToDX12Texture(
   PipelineStateCacheFormat formatFrom = PipelineStateCacheFormat::kBGR8;
   if (tensorDesc.channelType == kImageTensorChannelTypeRGB8) {
     formatFrom = PipelineStateCacheFormat::kRGB8;
-  } else if (inputDesc.Format == kImageTensorChannelTypeGRAY8) {
```

Comment:
Why does output format compare against DXGI formats (`outputDesc.Format == DXGI_FORMAT_R8_UNORM`), but inputDesc format compares against kImage enums? Should that be

`inputDesc.Format == DXGI_FORMAT_R8_UNORM`